playbook/app/pb_kits/playbook/pb_popover/index.js

```diff
@@ -36,33 +36,32 @@ export default class PbPopover extends PbEnhancedElement {
       }
 
       setTimeout(() => {
-        this.popper.update()
         this.toggleTooltip()
+        this.popper.update()
       }, 0)
     })
   }
 
   checkCloseTooltip() {
     document.querySelector('body').addEventListener('click', ({ target }) => {
-      const isTriggerElement = target.closest(`#${this.triggerElementId}`) !== null
       const isTooltipElement = target.closest(`#${this.tooltipId}`) !== null
 
       switch (this.closeOnClick) {
       case 'any':
         this.hideTooltip()
         break
       case 'outside':
-        if (!isTooltipElement || isTriggerElement) {
+        if (!isTooltipElement) {
           this.hideTooltip()
         }
         break
       case 'inside':
-        if (isTooltipElement || isTriggerElement) {
+        if (isTooltipElement) {
           this.hideTooltip()
         }
         break
       }
-    }, { once: true })
+    }, true)
```

Comment:
Do you still need this to only occur once? If so you can do, { once: true, capture: true }

Reply:
I'm actually not sure, It works either way, but I wonder if it is more performant to keep the { once: true }. @thestephenmarshall @jasperfurniss any thoughts?

Reply:
@smkopp92 I put it back